### ISSUE TITLE
Umlaute in name of HOME folder

### ISSUE THREAD

**Issue by honel** [mediathekview/plugin.video.mediathekview#96]
I am using Win10 and have Kodi (17.6) installed via Windows store.

Kodi works fine but for quite a while now, mediathekview is not working anymore. It seems to me that "Umlaute" in the directory name (e.g. if first/last name of the user contains an "Umlaut") are the source of the problem. Is there a chance this could be fixed?

`kodi.log:`
```
12:45:30.851 T:13956  NOTICE: special://profile/ is mapped to: special://masterprofile/
12:45:30.851 T:13956  NOTICE: -----------------------------------------------------------------------
12:45:30.851 T:13956  NOTICE: Starting Kodi (17.6 Git:20171114-a9a7a20). Platform: Windows NT x86 32-bit
12:45:30.851 T:13956  NOTICE: Using Release Kodi x32 build
12:45:30.851 T:13956  NOTICE: Kodi compiled Dec 10 2017 by MSVC 190024215 for Windows NT x86 32-bit version 10.0 (0x0A000000)
12:45:30.851 T:13956  NOTICE: Running on LENOVO 20266 with Windows 10, kernel: Windows NT x86 64-bit version 10.0
12:45:30.853 T:13956  NOTICE: FFmpeg version/source: ffmpeg-3.1-kodi
12:45:30.853 T:13956  NOTICE: Host CPU: Unknown, 4 cores available
12:45:30.853 T:13956  NOTICE: Desktop Resolution: 3200x1800 32Bit at 48Hz
12:45:30.853 T:13956  NOTICE: Running with restricted rights
12:45:30.864 T:13956  NOTICE: Aero is enabled
12:45:30.864 T:13956  NOTICE: special://xbmc/ is mapped to: C:\Program Files\WindowsApps\XBMCFoundation.Kodi_17.9.601.0_x86__4n2hpmxwrvr6p
12:45:30.864 T:13956  NOTICE: special://xbmcbin/ is mapped to: C:\Program Files\WindowsApps\XBMCFoundation.Kodi_17.9.601.0_x86__4n2hpmxwrvr6p
12:45:30.864 T:13956  NOTICE: special://xbmcbinaddons/ is mapped to: C:\Program Files\WindowsApps\XBMCFoundation.Kodi_17.9.601.0_x86__4n2hpmxwrvr6p/addons
12:45:30.864 T:13956  NOTICE: special://masterprofile/ is mapped to: C:\Users\Firstname Ölastname\AppData\Roaming\Kodi\userdata
12:45:30.864 T:13956  NOTICE: special://home/ is mapped to: C:\Users\Firstname Ölastname\AppData\Roaming\Kodi\
12:45:30.864 T:13956  NOTICE: special://temp/ is mapped to: C:\Users\Firstname Ölastname\AppData\Roaming\Kodi\cache
12:45:30.864 T:13956  NOTICE: special://logpath/ is mapped to: C:\Users\Firstname Ölastname\AppData\Roaming\Kodi\
12:45:30.864 T:13956  NOTICE: The executable running is: C:\Program Files\WindowsApps\XBMCFoundation.Kodi_17.9.601.0_x86__4n2hpmxwrvr6p\Kodi.exe
12:45:30.929 T:13956  NOTICE: Local hostname: io
12:45:30.929 T:13956  NOTICE: Log File is located: C:\Users\Firstname Ölastname\AppData\Roaming\Kodi\/kodi.log
12:45:30.929 T:13956  NOTICE: -----------------------------------------------------------------------
12:45:30.959 T:13956  NOTICE: load settings...
12:45:30.969 T:13956 WARNING: CSettingString: unknown options filler "timezonecountries" of "locale.timezonecountry"
12:45:30.969 T:13956 WARNING: CSettingString: unknown options filler "timezones" of "locale.timezone"
12:45:30.978 T:13956  NOTICE: No settings file to load (special://xbmc/system/advancedsettings.xml)
12:45:30.978 T:13956  NOTICE: No settings file to load (special://masterprofile/advancedsettings.xml)
12:45:30.978 T:13956  NOTICE: Default Video Player: VideoPlayer
12:45:30.978 T:13956  NOTICE: Default Audio Player: paplayer
12:45:30.978 T:13956  NOTICE: Disabled debug logging due to GUI setting. Level 0.
12:45:30.978 T:13956  NOTICE: Log level changed to "LOG_LEVEL_NORMAL"
12:45:30.979 T:13956  NOTICE: CMediaSourceSettings: loading media sources from special://masterprofile/sources.xml
12:45:30.980 T:13956  NOTICE: Loading player core factory settings from special://xbmc/system/playercorefactory.xml.
12:45:30.984 T:13956  NOTICE: Loaded playercorefactory configuration
12:45:30.984 T:13956  NOTICE: Loading player core factory settings from special://masterprofile/playercorefactory.xml.
12:45:30.984 T:13956  NOTICE: special://masterprofile/playercorefactory.xml does not exist. Skipping.
12:45:31.011 T:13956  NOTICE: Running database version Addons27
12:45:31.377 T:13956  NOTICE: ADDONS: Using repository repository.xbmc.org
12:45:31.470 T:4208  NOTICE: Found 2 Lists of Devices
12:45:31.470 T:4208  NOTICE: Enumerated DIRECTSOUND devices:
12:45:31.470 T:4208  NOTICE:     Device 1
12:45:31.470 T:4208  NOTICE:         m_deviceName      : {DAEA69B0-0FCE-4674-8D13-7372B9888E6D}
12:45:31.470 T:4208  NOTICE:         m_displayName     : Speakers - Lautsprecher (Realtek High Definition Audio)
12:45:31.470 T:4208  NOTICE:         m_displayNameExtra: DIRECTSOUND: Lautsprecher (Realtek High Definition Audio)
12:45:31.470 T:4208  NOTICE:         m_deviceType      : AE_DEVTYPE_PCM
12:45:31.470 T:4208  NOTICE:         m_channels        : FL,FR
12:45:31.470 T:4208  NOTICE:         m_sampleRates     : 48000
12:45:31.470 T:4208  NOTICE:         m_dataFormats     : AE_FMT_FLOAT
12:45:31.470 T:4208  NOTICE:         m_streamTypes     : No passthrough capabilities
12:45:31.470 T:4208  NOTICE:     Device 2
12:45:31.470 T:4208  NOTICE:         m_deviceName      : default
12:45:31.470 T:4208  NOTICE:         m_displayName     : default
12:45:31.470 T:4208  NOTICE:         m_displayNameExtra:
12:45:31.470 T:4208  NOTICE:         m_deviceType      : AE_DEVTYPE_PCM
12:45:31.470 T:4208  NOTICE:         m_channels        : FL,FR
12:45:31.470 T:4208  NOTICE:         m_sampleRates     : 48000
12:45:31.470 T:4208  NOTICE:         m_dataFormats     : AE_FMT_FLOAT
12:45:31.470 T:4208  NOTICE:         m_streamTypes     : No passthrough capabilities
12:45:31.470 T:4208  NOTICE: Enumerated WASAPI devices:
12:45:31.470 T:4208  NOTICE:     Device 1
12:45:31.470 T:4208  NOTICE:         m_deviceName      : {DAEA69B0-0FCE-4674-8D13-7372B9888E6D}
12:45:31.470 T:4208  NOTICE:         m_displayName     : Speakers - Lautsprecher (Realtek High Definition Audio)
12:45:31.470 T:4208  NOTICE:         m_displayNameExtra: WASAPI: Lautsprecher (Realtek High Definition Audio)
12:45:31.470 T:4208  NOTICE:         m_deviceType      : AE_DEVTYPE_PCM
12:45:31.470 T:4208  NOTICE:         m_channels        : FL,FR
12:45:31.470 T:4208  NOTICE:         m_sampleRates     : 192000,96000,48000,44100
12:45:31.470 T:4208  NOTICE:         m_dataFormats     : AE_FMT_S24NE4MSB,AE_FMT_S16NE,AE_FMT_S16LE,AE_FMT_S16BE
12:45:31.470 T:4208  NOTICE:         m_streamTypes     : No passthrough capabilities
12:45:31.470 T:4208  NOTICE:     Device 2
12:45:31.471 T:4208  NOTICE:         m_deviceName      : default
12:45:31.471 T:4208  NOTICE:         m_displayName     : default
12:45:31.471 T:4208  NOTICE:         m_displayNameExtra:
12:45:31.471 T:4208  NOTICE:         m_deviceType      : AE_DEVTYPE_PCM
12:45:31.471 T:4208  NOTICE:         m_channels        : FL,FR
12:45:31.471 T:4208  NOTICE:         m_sampleRates     : 192000,96000,48000,44100
12:45:31.471 T:4208  NOTICE:         m_dataFormats     : AE_FMT_S24NE4MSB,AE_FMT_S16NE,AE_FMT_S16LE,AE_FMT_S16BE
12:45:31.471 T:4208  NOTICE:         m_streamTypes     : No passthrough capabilities
12:45:31.631 T:13956  NOTICE: Found screen: Generic PnP Monitor on Intel(R) HD Graphics Family, adapter 0.
12:45:31.632 T:13956  NOTICE: Primary mode: 3200x1800@ 48.00 - Full Screen
12:45:31.633 T:13956  NOTICE: Additional mode: 320x200@ 48.00 - Full Screen
12:45:31.634 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.634 T:13956  NOTICE: Additional mode: 320x200@ 60.00 - Full Screen
12:45:31.635 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.635 T:13956  NOTICE: Additional mode: 320x240@ 48.00 - Full Screen
12:45:31.635 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.635 T:13956  NOTICE: Additional mode: 320x240@ 60.00 - Full Screen
12:45:31.636 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.636 T:13956  NOTICE: Additional mode: 400x300@ 48.00 - Full Screen
12:45:31.637 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.637 T:13956  NOTICE: Additional mode: 400x300@ 60.00 - Full Screen
12:45:31.637 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.637 T:13956  NOTICE: Additional mode: 512x384@ 48.00 - Full Screen
12:45:31.638 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.638 T:13956  NOTICE: Additional mode: 512x384@ 60.00 - Full Screen
12:45:31.639 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.639 T:13956  NOTICE: Additional mode: 640x400@ 48.00 - Full Screen
12:45:31.639 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.639 T:13956  NOTICE: Additional mode: 640x400@ 60.00 - Full Screen
12:45:31.640 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.640 T:13956  NOTICE: Additional mode: 640x480@ 48.00 - Full Screen
12:45:31.641 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.641 T:13956  NOTICE: Additional mode: 640x480@ 60.00 - Full Screen
12:45:31.641 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.641 T:13956  NOTICE: Additional mode: 800x600@ 48.00 - Full Screen
12:45:31.642 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.642 T:13956  NOTICE: Additional mode: 800x600@ 60.00 - Full Screen
12:45:31.643 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.643 T:13956  NOTICE: Additional mode: 1024x768@ 48.00 - Full Screen
12:45:31.643 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.643 T:13956  NOTICE: Additional mode: 1024x768@ 60.00 - Full Screen
12:45:31.644 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.644 T:13956  NOTICE: Additional mode: 1152x864@ 48.00 - Full Screen
12:45:31.644 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.644 T:13956  NOTICE: Additional mode: 1152x864@ 60.00 - Full Screen
12:45:31.645 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.645 T:13956  NOTICE: Additional mode: 1280x600@ 48.00 - Full Screen
12:45:31.646 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.646 T:13956  NOTICE: Additional mode: 1280x600@ 60.00 - Full Screen
12:45:31.646 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.646 T:13956  NOTICE: Additional mode: 1280x720@ 48.00 - Full Screen
12:45:31.647 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.647 T:13956  NOTICE: Additional mode: 1280x720@ 60.00 - Full Screen
12:45:31.648 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.648 T:13956  NOTICE: Additional mode: 1280x768@ 48.00 - Full Screen
12:45:31.648 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.648 T:13956  NOTICE: Additional mode: 1280x768@ 60.00 - Full Screen
12:45:31.649 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.649 T:13956  NOTICE: Additional mode: 1280x800@ 48.00 - Full Screen
12:45:31.650 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.650 T:13956  NOTICE: Additional mode: 1280x800@ 60.00 - Full Screen
12:45:31.650 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.650 T:13956  NOTICE: Additional mode: 1280x960@ 48.00 - Full Screen
12:45:31.651 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.651 T:13956  NOTICE: Additional mode: 1280x960@ 60.00 - Full Screen
12:45:31.652 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.652 T:13956  NOTICE: Additional mode: 1280x1024@ 48.00 - Full Screen
12:45:31.652 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.652 T:13956  NOTICE: Additional mode: 1280x1024@ 60.00 - Full Screen
12:45:31.653 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.653 T:13956  NOTICE: Additional mode: 1360x768@ 48.00 - Full Screen
12:45:31.654 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.654 T:13956  NOTICE: Additional mode: 1360x768@ 60.00 - Full Screen
12:45:31.654 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.654 T:13956  NOTICE: Additional mode: 1366x768@ 48.00 - Full Screen
12:45:31.655 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.655 T:13956  NOTICE: Additional mode: 1366x768@ 60.00 - Full Screen
12:45:31.656 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.656 T:13956  NOTICE: Additional mode: 1400x1050@ 48.00 - Full Screen
12:45:31.656 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.656 T:13956  NOTICE: Additional mode: 1400x1050@ 60.00 - Full Screen
12:45:31.657 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.657 T:13956  NOTICE: Additional mode: 1440x900@ 48.00 - Full Screen
12:45:31.658 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.658 T:13956  NOTICE: Additional mode: 1440x900@ 60.00 - Full Screen
12:45:31.658 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.658 T:13956  NOTICE: Additional mode: 1600x900@ 48.00 - Full Screen
12:45:31.659 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.659 T:13956  NOTICE: Additional mode: 1600x900@ 60.00 - Full Screen
12:45:31.660 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.660 T:13956  NOTICE: Additional mode: 1600x1200@ 48.00 - Full Screen
12:45:31.660 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.660 T:13956  NOTICE: Additional mode: 1600x1200@ 60.00 - Full Screen
12:45:31.661 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.661 T:13956  NOTICE: Additional mode: 1680x1050@ 48.00 - Full Screen
12:45:31.662 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.662 T:13956  NOTICE: Additional mode: 1680x1050@ 60.00 - Full Screen
12:45:31.662 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.662 T:13956  NOTICE: Additional mode: 1792x1344@ 48.00 - Full Screen
12:45:31.663 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.663 T:13956  NOTICE: Additional mode: 1792x1344@ 60.00 - Full Screen
12:45:31.664 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.664 T:13956  NOTICE: Additional mode: 1856x1392@ 48.00 - Full Screen
12:45:31.664 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.664 T:13956  NOTICE: Additional mode: 1856x1392@ 60.00 - Full Screen
12:45:31.665 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.665 T:13956  NOTICE: Additional mode: 1920x1080@ 48.00 - Full Screen
12:45:31.666 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.666 T:13956  NOTICE: Additional mode: 1920x1080@ 60.00 - Full Screen
12:45:31.666 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.666 T:13956  NOTICE: Additional mode: 1920x1200@ 48.00 - Full Screen
12:45:31.667 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.667 T:13956  NOTICE: Additional mode: 1920x1200@ 60.00 - Full Screen
12:45:31.668 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.668 T:13956  NOTICE: Additional mode: 1920x1440@ 48.00 - Full Screen
12:45:31.668 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.668 T:13956  NOTICE: Additional mode: 1920x1440@ 60.00 - Full Screen
12:45:31.669 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.669 T:13956  NOTICE: Additional mode: 2048x1152@ 48.00 - Full Screen
12:45:31.670 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.670 T:13956  NOTICE: Additional mode: 2048x1152@ 60.00 - Full Screen
12:45:31.670 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.670 T:13956  NOTICE: Additional mode: 2048x1536@ 48.00 - Full Screen
12:45:31.671 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.671 T:13956  NOTICE: Additional mode: 2048x1536@ 60.00 - Full Screen
12:45:31.672 T:13956  NOTICE: Previous line repeats 2 times.
12:45:31.672 T:13956  NOTICE: Additional mode: 3200x1800@ 48.00 - Full Screen
12:45:31.672 T:13956  NOTICE: Additional mode: 3200x1800@ 60.00 - Full Screen
12:45:31.672 T:13956  NOTICE: Checking resolution 15
12:45:32.486 T:8056  NOTICE: Running database version Addons27
12:45:32.488 T:8056  NOTICE: Running database version ViewModes6
12:45:32.491 T:8056  NOTICE: Running database version Textures13
12:45:32.495 T:8056  NOTICE: Running database version MyMusic60
12:45:32.500 T:8056  NOTICE: Running database version MyVideos107
12:45:32.503 T:8056  NOTICE: Running database version TV29
12:45:32.506 T:8056  NOTICE: Running database version Epg11
12:45:32.920 T:13956 WARNING: JSONRPC: Could not parse type "Setting.Details.SettingList"
12:45:33.129 T:13956  NOTICE: initialize done
12:45:33.129 T:13956  NOTICE: Running the application...
12:45:33.130 T:13956  NOTICE: starting upnp client
12:45:33.164 T:9108   ERROR: CUPnPDirectory::GetResource - unable to find uuid folder.jpg
12:45:33.392 T:9108  NOTICE: AddOnLog: PVR IPTV Simple Client: Loaded 759 channels.
12:45:36.046 T:9108  NOTICE: EPG::CEpgContainer::Start - EPG thread started
12:45:36.994 T:8528   ERROR: XFILE::CDirectory::GetDirectory - Error getting C:\Users\Firstname Ölastname\AppData\Roaming\Kodi\cache\plugin.video.youtube\
12:45:37.065 T:6372  NOTICE: AddOnLog: PVR IPTV Simple Client: EPG file path is not configured. EPG not loaded.
12:45:38.306 T:6128  NOTICE: [plugin.video.mediathekview-0.5.0:Service]: Init (instance id: 11daa51ca702b718f331e7e42a9160f5)
12:45:38.308 T:6128  NOTICE: [plugin.video.mediathekview-0.5.0:Service]: Found other instance with id 11daa51ca702b718f331e7e42a9160f5
12:45:38.308 T:6128  NOTICE: [plugin.video.mediathekview-0.5.0:Service]: Startup delayed by 1 second(s) waiting the other instance to shut down
12:45:39.320 T:6128  NOTICE: [plugin.video.mediathekview-0.5.0:Updater]: Database driver: Internal (sqlite)
12:45:39.320 T:6128  NOTICE: [plugin.video.mediathekview-0.5.0:StoreSQLite]: Using SQLite version 3.10.2, python library sqlite3 version 2.6.0
12:45:39.329 T:6128   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.WindowsError'>
                                            Error Contents: (3, 'Das System kann den angegebenen Pfad nicht finden', 'C:\\Users\\Firstname \xc3\x96lastname\\AppData\\Roaming\\Kodi\\userdata\\addon_data\\plugin.video.mediathekview\\')
                                            Traceback (most recent call last):
                                              File "C:\Users\Firstname ?astname\AppData\Roaming\Kodi\addons\plugin.video.mediathekview\service.py", line 101, in <module>
                                                service.Init()
                                              File "C:\Users\Firstname ?astname\AppData\Roaming\Kodi\addons\plugin.video.mediathekview\service.py", line 56, in Init
                                                self.updater.Init( convert = True )
                                              File "C:\Users\Firstname ?lastname\AppData\Roaming\Kodi\addons\plugin.video.mediathekview\resources\lib\updater.py", line 58, in Init
                                                self.db.Init( convert = convert )
                                              File "C:\Users\Firstname ?astname\AppData\Roaming\Kodi\addons\plugin.video.mediathekview\resources\lib\store.py", line 28, in Init
                                                return self.db.Init( reset, convert )
                                              File "C:\Users\Firstname ?astname\AppData\Roaming\Kodi\addons\plugin.video.mediathekview\resources\lib\storesqlite.py", line 35, in Init
                                                os.mkdir( self.settings.datapath )
                                            WindowsError: (3, 'Das System kann den angegebenen Pfad nicht finden', 'C:\\Users\\Firstname \xc3\x96lastname\\AppData\\Roaming\\Kodi\\userdata\\addon_data\\plugin.video.mediathekview\\')
                                            -->End of Python script error report<--
12:45:39.342 T:6128 WARNING: CPythonInvoker(2, C:\Users\Firstname Ölastname\AppData\Roaming\Kodi\addons\plugin.video.mediathekview\service.py): the python script "C:\Users\Firstname Ölastname\AppData\Roaming\Kodi\addons\plugin.video.mediathekview\service.py" has left several classes in memory that we couldn't clean up. The classes include: class XBMCAddon::xbmcaddon::Addon,class XBMCAddon::xbmcaddon::Addon,class XBMCAddon::xbmcaddon::Addon,class XBMCAddon::xbmcaddon::Addon,class PythonBindings::XBMCAddon_xbmc_Monitor_Director
12:45:42.952 T:13956  NOTICE: Storing total System Uptime
12:45:42.952 T:13956  NOTICE: Saving settings
12:45:42.955 T:13956  NOTICE: Saving skin settings
12:45:42.957 T:13956  NOTICE: stop all
12:45:42.957 T:13956  NOTICE: stop player
12:45:42.957 T:13956  NOTICE: stopping upnp
12:45:43.393 T:13956  NOTICE: clean cached files!
12:45:43.429 T:13956  NOTICE: closing down remote control service
12:45:43.429 T:13956  NOTICE: stopped
12:45:43.630 T:13956  NOTICE: destroy
12:45:43.630 T:13956  NOTICE: unload skin
12:45:43.737 T:13956  NOTICE: unload sections
12:45:43.740 T:13956  NOTICE: special://profile/ is mapped to: special://masterprofile/
12:45:43.741 T:13956  NOTICE: PVRManager - stopping
12:45:43.774 T:13956  NOTICE: application stopped...
```

**Comment by tuxpoldo** [comment]
Geeee! What a mess! This is a really hard one. I will try to investigate this, but unfortunately this is one of the many Unicode-Python-2-Shitty issues...

**Comment by tuxpoldo** [comment]
In the next release I have to implement Issue #116 - maybe this may solve also this problem? Let's hope....

**Comment by codingPF** [comment]
Released to official Kodi Repo in v0.6.4
